orders/views.py

```diff
@@ -74,4 +75,31 @@ def get(self, request):
                 order__order_status_id=OrderStatus.BASKET
             )]
 
-        return JsonResponse({'message':'SUCCESS', 'items_in_cart':order_items}, status=200)
\ No newline at end of file
+        return JsonResponse({'message':'SUCCESS', 'items_in_cart':order_items}, status=200)
+
+    @login_decorator
+    def patch(self, request):
+        try:
+            data          = json.loads(request.body)
+            order_item_id = data['order_item_id']
+            count         = data.get('count')
+            select        = data.get('select')
```

Comment:
```suggestion
            count         = data['count']
            select        = data['select']
```
무조건 필수로 받는 것은 어떨까요? 정보를 수정하는 거니까 수정해야 하는 정보들은 필수로 받으면 좋을 것 같아요.
필수로 받게 되면서 밑에 if 들이 어떻게 바뀔 수 있을지 생각해 보고 수정해 보세요.